todoapp/src/components/todo.tsx

```diff
@@ -0,0 +1,264 @@
+    }
+  },
+  container: {
+    "&&": {
+      backgroundColor: '#ff5252',
+      color: 'white',
+      padding: '1rem',
+      width: '30%',
+      height: '70vh',
+      boxShadow: '5px 5px 25px -5px rgba(0,0,0,0.5)',
+      borderRadius: 0,
+      margin: 'auto',
+      textAlign: 'left',
+      alignItems: 'center',
+      justifyContent: 'space-between',
+      position: 'relative',
+      "@media (min-width: 768px)": {
+        width: "40%",
+      },
+      "@media (min-width: 1592px)": {
+        width: "30%",
+      },
+    }
+
+  },
+  firststack: {
+    "&&": {
+      marginBottom: '40px'
+    }
+  },
+  headings: {
+    "&&": {
+      marginLeft: '50px',
+      marginTop: '15px',
+      fontWeight: 'bold',
+    }
+  },
+  subheadings: {
+    "&&": {
+      marginLeft: '60px',
+      mb: '15px',
+      fontWeight: 'bold',
+    }
+  },
+  hr: {
+    "&&": {
+      margin: 'center',
+      marginInlineStart: '7%',
+      marginInlineEnd: '7%',
+    }
+  },
+  card: {
+    "&&": {
+      background: '#ff5252',
+      maxHeight: '300px',
+      overflowY: 'auto',
+    }
+  },
+  cardContent: {
+    "&&": {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'space-between',
+      background: 'rgba(255, 205, 210, 0.25)',
+    }
+  },
+  stacktwo: {
+    "&&": {
+      position: "absolute",
+      width: "100%",
+      left: 0,
+    }
+  },
+  taskName: {
+    "&&": {
+      color: "white",
+      marginLeft: "50px",
+    }
+  },
+  newTodo: {
+    "&&": {
+      position: 'absolute',
+      bottom: '50px',
+      width: '100%',
+      flexDirection: 'column',
+      marginLeft: '60px',
+      marginRight: '50px',
+      overflowY: 'auto',
+    }
+  },
+  todoHeading: {
+    "&&": {
+      mb: '5px',
+      fontWeight: 'bold',
+    }
+  },
+  todoContainer: {
+    "&&": {
+      gap: '0.4rem',
+      marginRight: '60px',
+      display: 'flex',
+    }
+  },
+  todoTextField: {
+    "&&": {
+      width: '60%',
+      fontWeight: 'bold',
+      borderRadius: 0,
+      '& .MuiOutlinedInput-root': {
+        backgroundColor: 'white',
+        '&:hover fieldset': {
+          borderColor: 'white',
+        },
+        '&.Mui-focused fieldset': {
+          borderColor: '#ff5252',
+        },
+      },
+      "@media (max-width: 600px)": {
+        todoTextField: {
+          width: '50%',
+        }
+      }
+    }
+  },
+  todoButton: {
+    "&&": {
+      backgroundColor: '#ff5252',
+      borderColor: 'white',
+      color: 'white',
+      fontWeight: 'bold',
+      borderRadius: 0,
+      '&:hover': {
+        backgroundColor: 'black',
+      },
+      '&:focus': {
+        outlineColor: '#ff5252',
+      },
+      "@media (max-width: 600px)": {
+        todoButton: {
+          width: '100%',
+        }
+      }
+    }
+  }
+})
+
+
+function Todos(): JSX.Element {
+  const isSmallScreen = useMediaQuery('(max-width:600px)');
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
```

Comment:
remember we are not using <div> or anything outside of Mui. If possible remove this div, why we need two root level div or boxes.